README.md

```diff
@@ -101,30 +252,117 @@ options:
--- <a href="https://twitter.com/JustinClarkJO">Twitter</a>
--- <a href="https://www.linkedin.com/in/justin-w-clark/">Linkedin</a>
+## Output files & spreadsheets
 
-Or email me at jclarksummit@gmail.com
+Wayback Google Analytics allows you to export your findings to either `.csv` or `.xlsx` spreadsheets. When choosing to save your findings as a spreadsheet, the tool generates two databases: one where each url is the primary index and another where each identified code is the primary index. In an `.xlsx` file this is one spreadsheet with two sheets, while the `.csv` option generates one file sorted by codes and another sorted by websites. All output files can be found in `/output`, which is created in the directory from which the code is executed.
 
-# Further Reading
+#### Example spreadsheet
+
+Let's say we're looking into data from 4 websites from 2015 until present and we want to save what we find in an excel spreadsheet. Our start command looks something like this:
+
+```terminal
+wayback-google-analytics -u https://yapatriot.ru https://zanogu.com https://whoswho.com.ua https://adamants.ru -s 01/01/2015 -f yearly -o xlsx
+```
+
+The result is a single `.xlsx` file with two sheets.
+
+Ordered by website:
+<div align="center">
+  <img src="./docs/imgs/xlsxbysite.png" width=1000>
+</div>
+
+Ordered by code:
+<div align="center">
+  <img src="./docs/imgs/xlsxbycode.png">
+</div>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+### Bugs and feature requests
+
+Please feel free to [open an issue](https://github.com/bellingcat/wayback-google-analytics/issues) should you encounter any bugs or have suggestions for new features or improvements. You can also [reach out to me](#contact) directly with suggestions or thoughts.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- DEVELOPMENT -->
+## Development
+
+### Testing
+
+* Run tests with `python -m unittest discover`
+* Check coverage with `coverage run -m unittest`
+
+### Using Poetry for Development
+
+Wayback Google Analytics uses [Poetry](https://python-poetry.org/), a Python dependency management and packaging tool. A github workflow automates the publishing process on pushes to main ((see our workflow here)[https://github.com/bellingcat/wayback-google-analytics/actions]), but be sure to update the version number in `pyproject.toml` when updating.
+
+To publish a new version to PyPI, you can follow these steps:
+
+1. Change the version number in [pyproject.toml](pyproject.toml)
+2. Create a new tag for that version `git tag "vX.0.0"`
+3. Push the tag `git push --tags`
+
+<!-- LICENSE -->
+## License
+
+Distributed under the MIT License. See `LICENSE.txt` for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTACT -->
+## Contact
+
+You can contact me through email or social media.
+
+* email: jclarksummit@gmail.com
```

Comment:
```suggestion
* email: jclarksummit at gmail dot com
```
not sure how publicly widespread this email address is, but GH is easy to scrape and can easily be used for automated phishing campaigns. do disregard if that's not a concern